src/utilities/request.jl

```diff
@@ -190,7 +191,7 @@ function submit_request(aws::AbstractAWSConfig, request::Request; return_headers
         return false
     end
 
-    delays = AWSExponentialBackoff(; max_attempts=3)
+    delays = AWSExponentialBackoff(; max_attempts=max_attempts(aws))
```

Comment:
How about making the whole delay strategy configurable?

```suggestion
    delays = delays(aws)
```

to give more control, e.g. tweak the initial delay or delay between retries.

Perhaps this would require exporting `AWSExponentialBackoff` so it's easy for users to tweak only the max attempt.